### ISSUE TITLE
Simplify initialization of Each rule - pass array of rules instead of RuleSet

### ISSUE THREAD

**Issue by arogachev** [yiisoft/validator#179]
Example uses changes introduced in #175.

Before:

```php
new Each(new RuleSet([new Number(min: 0, max: 255)]))
```

After:

```php
new Each([new Number(min: 0, max: 255)])
```

`RuleSet` is always expected, but the latter is a bit more readable. What do you think?

**Comment by arogachev** [comment]
Formatter can be passed to `Each` instead.

**Comment by arogachev** [comment]
Decided to fix it in #175 due to possible large amount of merge conflicts in the future.
